blog/_posts/2020-06-03-systematics-blogpost.md

```diff
@@ -0,0 +1,335 @@
+          
+          if((percentile_data2[GenValue][j] <= PhyloDiversity) && (percentile_data2[GenValue][j + 1] > PhyloDiversity)){
+            std::cout << "phylo diversity is in between: " << percentile_data2[GenValue][j] << "and " << percentile_data2[GenValue][j+1] << std::endl;
+            std::cout << "I is equal to: " << GenValue << std::endl; 
+            std::cout << "J is equal to: " << j << std::endl;
+
+            std::cout << "The Phylogentic diversity value " << PhyloDiversity << " is in the " << j << " percentile, in the " << ((GenValue + 1)* 10) << " generation" << std::endl;  
+
+            fs << ((GenValue + 1)* 10) << "," << j << std::endl; 
+
+            percentFound = true; 
+          }
+          
+          if(PhyloDiversity >= percentile_data2[GenValue][lastval]){ 
+              fs << ((GenValue + 1) * 10) << "," << 100 << std::endl; 
+              fs.close(); 
+            }
+
+        if(percentFound == true){ 
+            break; 
+          }
+          }
+          if(percentFound == false){ 
+            std::cout << "PHYLO DIVERSITY IS IN 100TH PERCENTILE" << std::endl; 
+           }
+        }
+```
+
+------------------------- 
+## **Method**
+
+With each of the three models, I ran each one 1000 times for every 10 generations and collected the phylogenetic diversity values at the end of each run.
+
+For example, I would set the number of generations in the null model to 10. Then, I would run it 1000 times, and at the end of each run, record the final diversity. I would then do 20 generations, and so forth, all the way through 100 generations. 
+
+After the initial data collection, I took that data and ran it through a python script which created a percentile range. It did this by sorting all of the data from least to greatest. It would then take every 10th value in the dataset, to output 100 final diversity values, each corresponding to a percentile value from 0 to 100. For each of the different models, I repeated the same process.
+
+To incorporate this data into the systematics manager, I imported the percentile csv files into the two functions described in the systematics section. Wherever these functions are called in future code, they will calculate the phylogenetic diversity of the tree and return the percentile value for how the tree compares to the models. 
+
+After I had this framework setup I decided to test its reliability. I ran my models once again and had the systematics manager classify each tree's final phylogenetic diversity after each set of generations. I used the file containing percentiles for the tree that used mutations but had no pressure for diversity. 
+
+When I used this process on the tree with mutations but with no pressure for diversity, I would expect to see values in the 50th percentile range. However, if repeated for the tree with a pressure for diversity, I would then expect values near the 100th percentile. 
```

Comment:
I'd suggest "I would then expect values significantly higher than the 50th percentile" since it's totally possible for there to be a meaningful effect without it being that dramatic